include/config.h

```diff
@@ -93,8 +98,8 @@
 #define HANGING_FIX
 // The last frame that will be considered a firsty when wallkicking
 #define FIRSTY_LAST_FRAME 1
-// 46 degree walkicks
-//#define WALLKICKS_46_DEGREES
+// The maximum angle the player can wall kick, in degrees. 0..90
+#define WALL_KICK_DEGREES 45
```

Comment:
```suggestion
// The maximum angle the player can wall kick, in degrees. 0..90. To allow 45 degree wall kicks, you must supply `46` to allow 45 and under
#define WALL_KICK_DEGREES 45
```
also its wild but this works with 180 lmao

Reply:
agreed